index.bs

```diff
@@ -1819,7 +1819,7 @@ a numbered step. If outdented, it (today) is rendered either as a bullet in the
                         ::  |attestationObject|
 
                         :   {{AuthenticatorAttestationResponse/[[transports]]}}
-                        ::  A sequence of zero or more unique {{DOMString}}s, in lexicographical order, that the |authenticator| is believed to support. The values SHOULD be members of {{AuthenticatorTransport}}.
+                        ::  A sequence of zero or more unique {{DOMString}}s, in lexicographical order, that the |authenticator| is believed to support. The values SHOULD be members of {{AuthenticatorTransport}}, but [=client platforms=] MUST ignore unknown values.
```

Comment:
Isn't this value set by the client? Then it seems a bit odd that the client would be the one to ignore unknown values, but I'm not sure what else would make sense to say here.

Reply:
It's definitely awkward, but an authenticator could betray the spec and return an unknown value, which the system needs to handle.

Reply:
Yeah, that makes sense under the assumption that the client asks the authenticator to generate this value, but it's not really evident here that that is the expected case... Anyway, this is probably fine, just something to consider.